src/Operation/CreateCollection.php

```diff
@@ -167,6 +179,10 @@ public function __construct($databaseName, $collectionName, array $options = [])
             throw InvalidArgumentException::invalidType('"storageEngine" option', $options['storageEngine'], 'array or object');
         }
 
+        if (isset($options['timeseries']) && ! is_array($options['timeseries']) && ! is_object($options['timeseries'])) {
```

Comment:
Reading this again with fresh eyes, I find it very odd that the server team didn't use camelCase for this option name (vs. `expireAfterSeconds`).